### ISSUE TITLE
Virtualized list elements are inaccessible via keyboard

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/uikit#1343]
https://preview.gravity-ui.com/uikit/?path=/story/unstable-uselist-virtualizedlist--virtualized-list

The elements should be accessible via keyboard, as in List.

**Comment by amje** [comment]
Cannot reproduce
